CppCustomVisualizer2/headers/TargetApp.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <vector>
+
+class Sample
+{
+public:
+    // a.size() == b.size()
```

Comment:
Does this comment need to be here? Or should this be an assert somewhere else.

Reply:
There's a check in the visualizer. Maybe I can change this comment to something like `Expecting a.size() == b.size()`?